src/webgl/p5.RendererGL.js

```diff
@@ -932,13 +932,20 @@ class RendererGL extends Renderer {
     const defaultId = this.canvas.id;
     const isPGraphics = this._pInst instanceof Graphics;
 
+    // Preserve existing position and styles before recreation
+    const prevStyle = {
+      position: this.canvas.style.position,
+      top: this.canvas.style.top,
+      left: this.canvas.style.left,
+    };
+
     if (isPGraphics) {
       const pg = this._pInst;
       pg.canvas.parentNode.removeChild(pg.canvas);
       pg.canvas = document.createElement("canvas");
       const node = pg._pInst._userNode || document.body;
       node.appendChild(pg.canvas);
-      Element.call(pg, pg.canvas, pg._pInst);
+      p5.Element.call(pg, pg.canvas, pg._pInst);
```

Comment:
Does this still work if we take off the p5 prefix here too as it was before?

```suggestion
      Element.call(pg, pg.canvas, pg._pInst);
```

Reply:
Yes, I tested it after removing the p5. prefix, and everything works correctly. The canvas still preserves its position after calling noSmooth().